en/04_Changelogs/5.0.0.md

```diff
@@ -125,90 +190,90 @@ This is a major release and as a result there are a number of breaking API chang
+- [`Extensible::invokeWithExtensions()`](api:SilverStripe\Core\Extensible::invokeWithExtensions()) and [`Extensible::extend()`](api:SilverStripe\Core\Extensible::extend()) now use the splat operator instead of having a concrete number of possible arguments.
+- The constructor for [`Versioned`](api:SilverStripe\Versioned\Versioned::__construct()) now explicitly only accepts `$mode` as a single argument.
 - The following classes have been moved from `silverstripe/cms` to `silverstripe/admin` and renamed:
-  - `SilverStripe\CMS\Controllers\SilverStripeNavigator` is now in silverstripe/admin as `SilverStripe\Admin\Navigator\SilverStripeNavigator`
-  - `SilverStripe\CMS\Controllers\SilverStripeNavigatorItem` is now in silverstripe/admin as `SilverStripe\Admin\Navigator\SilverStripeNavigatorItem`
-  - `SilverStripe\CMS\Controllers\SilverStripeNavigatorItem_Unversioned` is now in silverstripe/admin as `SilverStripe\Admin\Navigator\SilverStripeNavigatorItem_Unversioned`
+  - `SilverStripe\CMS\Controllers\SilverStripeNavigator` is now in `silverstripe/admin` as `SilverStripe\Admin\Navigator\SilverStripeNavigator`
+  - `SilverStripe\CMS\Controllers\SilverStripeNavigatorItem` is now in `silverstripe/admin` as `SilverStripe\Admin\Navigator\SilverStripeNavigatorItem`
+  - `SilverStripe\CMS\Controllers\SilverStripeNavigatorItem_Unversioned` is now in `silverstripe/admin` as `SilverStripe\Admin\Navigator\SilverStripeNavigatorItem_Unversioned`
 - The following classes have been moved from `silverstripe/cms` to `silverstripe/versioned-admin` and renamed:
-  - `SilverStripe\CMS\Controllers\SilverStripeNavigatorItem_ArchiveLink` is now in silverstripe/versioned as `SilverStripe\VersionedAdmin\Navigator\SilverStripeNavigatorItem_ArchiveLink`
-  - `SilverStripe\CMS\Controllers\SilverStripeNavigatorItem_LiveLink` is now in silverstripe/versioned as `SilverStripe\VersionedAdmin\Navigator\SilverStripeNavigatorItem_LiveLink`
-  - `SilverStripe\CMS\Controllers\SilverStripeNavigatorItem_StageLink` is now in silverstripe/versioned as `SilverStripe\VersionedAdmin\Navigator\SilverStripeNavigatorItem_StageLink`
-- Various internal controller methods have been updated to now consistently return an `SilverStripe\Control\HTTPResponse` object, rather than an undeclared `mixed` return type. These updates do not affect custom controller action methods which can still have an undeclared mixed return type.
+  - `SilverStripe\CMS\Controllers\SilverStripeNavigatorItem_ArchiveLink` is now in `silverstripe/versioned` as `SilverStripe\VersionedAdmin\Navigator\SilverStripeNavigatorItem_ArchiveLink`
+  - `SilverStripe\CMS\Controllers\SilverStripeNavigatorItem_LiveLink` is now in `silverstripe/versioned` as `SilverStripe\VersionedAdmin\Navigator\SilverStripeNavigatorItem_LiveLink`
+  - `SilverStripe\CMS\Controllers\SilverStripeNavigatorItem_StageLink` is now in `silverstripe/versioned` as `SilverStripe\VersionedAdmin\Navigator\SilverStripeNavigatorItem_StageLink`
```

Comment:
Didn't addapi links here because otherwise you wouldn't be able to see the FQN of the classes.

Reply:
You do realise you can just put the FQCN in the link right? Like [`SilverStripe\VersionedAdmin\Navigator\SilverStripeNavigatorItem_ArchiveLink`](api:SilverStripe\VersionedAdmin\Navigator\SilverStripeNavigatorItem_ArchiveLink)

So if that's the only reason, we should add the link. If you didn't want to do it because it gets messy (which is what I understood from our chat yesterday), then that's totally fine.